src/components/Buttons/Button.js

```diff
@@ -207,7 +207,7 @@ const getSnacksStyles = props => {
 const Button = props => {
   const snacksStyles = getSnacksStyles(props)
 
-  const ElementType = props.href ? 'a' : 'button'
+  const ElementType = props.customElement ? props.customElement : props.href ? 'a' : 'button'
```

Comment:
we should call the props `elementType` to be consistent with other snacks component APIs